packages/multi-select-combo-box/theme/lumo/vaadin-multi-select-combo-box-chip-styles.js

```diff
@@ -52,28 +44,28 @@ const chip = css`
     border-color: var(--lumo-contrast-30pct);
   }
 
-  :host([part~='overflow'])::before {
+  :host([slot='overflow'])::before {
     left: calc(-1 * var(--lumo-space-s) / 2);
   }
 
-  :host([part~='overflow'])::after {
+  :host([slot='overflow'])::after {
     left: calc(-1 * var(--lumo-space-s));
   }
 
-  :host([part~='overflow-two']) {
+  :host([label='2']) {
```

Comment:
It looks like we mostly need this internally to make the overflow chip display as a "stack", so for that purpose using the label attribute is fine.

If we intend for developers to style this, then having to use the label attribute as selector is a  bit weird. It seems like we could use any other attribute like `count` or `item-count` to make this more semantic.

Reply:
Yes, I was thinking about that - abusing `label` feels wrong indeed. Will update to use `count` as you suggested.

Reply:
Changed to set `count` attribute.